get_deps.py

```diff
@@ -30,14 +30,23 @@ def main():
 		if dll_name.startswith('???'):
 			print('Unknown DLL?')
 			continue
-		if dll_path.lower().startswith('/c/windows') or dll_path.lower().startswith('c:/windows'):
+		try:
+			# ldd on msys2 gives Unix-style paths, but Python wants them Windows-style
+			# Try cygpath to convert them
+			dll_path = subprocess.check_output(['cygpath', '-w', dll_path]).strip()
+		except:
+			# cygpath doesn't exist or failed. Carry on and hope for the best
+			pass
+		if dll_path.lower().startswith('c:\\windows'):

```

Comment:
heh, damnit.. now you might have 'c:\\windows'  (if cygpath succeeded) or 'c:/windows' (if cygpath failed). Maybe python os path can fix it?

Anyhow this feels like bikeshedding.

Reply:
Yeah, those issues are probably best dealt with when they appear. Because you have to build xqemu with msys2, I'm not convinced we're ever in the situation where cygpath isn't available. Not to mention you have to run the script in msys2 anyway, otherwise ldd isn't available.

Reply:
nit: [`os.path.normcase(path)` could be used](https://docs.python.org/2/library/os.path.html#os.path.normcase) instead of the .lower() on every use and it would solve this issue.

Reply:
Hm, normcase replaces backslashes by forward slashes on msys, which goes against the docs. The offical Python release for Windows behaves as documented though. I'll hold off on this for now, because it doesn't seem to improve the situation.